pkg/configuration/testdata/tnf_test_config.yml

```diff
@@ -29,3 +29,6 @@ skipScalingTestStatefulSetNames:
 validProtocolNames:
   - "http3"
   - "sctp"
+ServicesIgnoreList:
+  - "hazelcast-platform-controller-manager-service"
+  - "hazelcast-platform-webhook-service"
```

Comment:
Add the new line at the end of the file:
 https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
fixed